set.mm

```diff
@@ -23303,6 +23303,13 @@ correct definition apparently has never been published ( ` E* ` means
 
   ${
     $d x y $.
+    $( Two ways to express "exactly one thing exists".  Both sides are false in
+       set theory; see theorems ~ neutru and ~ dtru .  (Contributed by BJ,
+       7-Oct-2022.) $)
+    exists0 $p |- ( E! x T. <-> A. x x = y ) $=
+      ( wtru weu weq wb wal wex bicom tbtru bitr4i albii exbii nfae 19.9 3bitri
+      eu6 ) CADCABEZFZAGZBHRAGZBHUACABQTUABSRASRCFRCRIRJKLMUABABBNOP $.
+
     $( Two ways to express "exactly one thing exists".  The left-hand side
```

Comment:
exists1 should be reproven along the lines equid bitru exists0 to avoid proof duplications.

Reply:
Will do. As for labels, actually there was already exists1 and exists2, so it was simple numbering rather than the "1" for "unique". But I'm open to change labels. Since they are not used later, we can have longer labels. `exeu` ? `exuniq(ue)` ?

Reply:
Or `eutruae` since `eutru` is LHS and `ae` is RHS.  Maybe @avekens has an opinion on that ?

Reply:
I chose `euae` in the latest commit. Also, I kept Norm as a contributor and I put myself only as "revised by", so that Norm's contribution stays even in case exists1 is removed.